x-pack/plugins/observability/public/pages/overview/index.tsx

```diff
@@ -39,6 +40,11 @@ function calculateBucketSize({ start, end }: { start?: number; end?: number }) {
 export function OverviewPage({ routeParams }: Props) {
   useTrackPageview({ app: 'observability-overview', path: 'overview' });
   useTrackPageview({ app: 'observability-overview', path: 'overview', delay: 15000 });
+
+  // TODO: figure out if this is really necessary? If I don't add this then navigating from cases to
+  // the overview page leaves the glasses icon in the header
+  useKibana().services.chrome.setBadge();
```

Comment:
@stephmilovic @smith I'm seeing an issue where if I navigate to the cases overview page in observability and then navigate to the observability overview page the glasses header icon stays there when it shouldn't. I'm not sure if this is related to the other breadcrumbs issue described here: https://github.com/elastic/kibana/issues/102131

Any thoughts?

Reply:
i'd defer to @smith on this `¯\_(ツ)_/¯`

Reply:
I don't think this is related to the breadcrumbs, but it shouldn't be called in the main render body of the component.

I think the correct way to do this would be to call `setBadge` in a `useEffect` and return a function from the effect that calls `setBadge(undefined)` so the badge is removed when the component is unmounted.

Reply:
Oh I see, you're using an effect in `HeaderBadge`, I think you need to make that one look like:

```js
useEffect(() => {  
  setBadge();
  return () => { chrome.setBadge(undefined); }
}, [chrome.setBadge, setBadge]);
```